media/css/base/menu-resp.less

```diff
@@ -186,11 +185,6 @@ html[dir="rtl"] #nav-main .submenu {
         padding: 0;
     }
 
-    .js #nav-main li .submenu {
-        xleft: 80px;
-        xtop: auto;
```

Comment:
Wow, where did this even come from? Did I do that? Probably been here for years...